docs/overview.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+The Foreman project runs a number of different servers for testing, packaging, and continuous integration.
+
+## Access
+
+Access to Foreman project infrastructure is available for those who wish to assist in building packages, testing, and building Jenkins jobs. Fork http://github.com/theforeman/foreman-infra and add an ssh_user resource to the users module (see puppet/modules/users/manifests/init.pp) and your key into the files directory. Submit a pull request to the infrastructure project and then talk to ? in #theforeman-dev on irc.freenode.net. One of them can merge your change and update the puppetmaster.
```

Comment:
```suggestion
Access to Foreman project infrastructure is available for those who wish to assist in building packages, testing, and building Jenkins jobs.

Fork https://github.com/theforeman/foreman-infra and add your key into the [files directory](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/users/files) of the [`users` module](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/users/). Submit a pull request to the infrastructure project and then talk to ? in #theforeman-dev on irc.freenode.net. One of them can merge your change and add your user in the Foreman web UI.
```

Reply:
Perhaps a Discourse post is a better suggestion?

Reply:
Also that, yeah.